CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 
 * [#4746](https://github.com/bbatsov/rubocop/pull/4746): The `Lint/InvalidCharacterLiteral` cop has been removed since it was never being actually triggered. ([@deivid-rodriguez][])
 * [#4789](https://github.com/bbatsov/rubocop/pull/4789): Analyzing code that needs to support MRI 1.9 is no longer supported. ([@deivid-rodriguez][])
+* [#4787](https://github.com/bbatsov/rubocop/pull/4787): Analyzing code that needs to support MRI 2.0 is no longer suupported. ([@deivid-rodriguez][])
+* [#4787](https://github.com/bbatsov/rubocop/pull/4787): RuboCop no longer installs on MRI 2.0. ([@deivid-rodriguez][])
```

Comment:
I think this should be "Drop support for Ruby 2.0".

Reply:
You mean compressing the two previous entries into one, or replacing MRI with Ruby in all the related entries, or both?